packages/components/psammead-leading-story-promo/src/index.jsx

```diff
@@ -0,0 +1,89 @@
+import React from 'react';
+import styled, { css } from 'styled-components';
+import { node } from 'prop-types';
+import { grid } from '@bbc/psammead-styles/detection';
+import { GEL_GROUP_4_SCREEN_WIDTH_MIN } from '@bbc/gel-foundations/breakpoints';
+import { GEL_SPACING_DBL } from '@bbc/gel-foundations/spacings';
+
+const fourOfSixColumnsMaxWidthScaleable = `66.67%`;
+// (4 / 6) * 100 = 66.6666666667 = 66.67%
+
+const twoOfSixColumnsMaxWidthScaleable = `33.33%`;
+// (2 / 6) * 100 = 0.3333333333 = 33.33%
+
+const LeadingPromoWrapper = styled.div`
+  position: relative;
```

Comment:
Why do we need to set the position to relative?

Reply:
This is not necessary. Removed it

Reply:
I think this `relative` prevents the link's `:before` from taking up the whole screen as can be seen here: 
![image](https://user-images.githubusercontent.com/43134742/67371641-98cb9100-f574-11e9-8880-b58ade54aad8.png)
I don't know how it acts when integrated into Simorgh though.